src/compas_fab/backends/kinematics/solvers/spherical_wrist.py

```diff
@@ -174,7 +174,10 @@ def inverse_kinematics_spherical_wrist(target_frame, points):
         elbow_frame = Frame(p1A, elbow_dir, [0, 0, 1])
         elbow_plane = (p1A, elbow_frame.normal)
 
-        _, (center, radius, normal) = intersection_sphere_sphere(sphere1, sphere2)
+        sphere1_prim = (sphere1.base, sphere1.radius)
+        sphere2_prim = (sphere2.base, sphere2.radius)
+
+        _, (center, radius, normal) = intersection_sphere_sphere(sphere1_prim, sphere2_prim)
```

Comment:
Is this due to a change in compas? Do we need to correct/limit the required version?

Reply:
👍